### PR TITLE
Winners model

### PR DESCRIPTION
**Pull request by Shargrove09**
Add more relations to better track winner of sets and matches. 
Fix some inconsistencies in schema

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -29,6 +29,8 @@ model Player {
   playerName     String          @map("player_name")
   playerSessions PlayerSession[]
   playerStats    PlayerStat[]
+  matchWins      Match[]         @relation("matchWinner")
```
Comment (tydolla00) [inline prisma/schema.prisma:32]: Don't forget map _ for camelCases

**Review by tydolla00 — CHANGES_REQUESTED** [review]
add _s to db model

**Review by Shargrove09 — COMMENTED** [review]
See text regarding _s

File: prisma/schema.prisma [hunk 2]
```diff
@@ -68,19 +72,22 @@ model Session {
...
+  sessionId      Int             @map("session_id")
+  session        Session         @relation(fields: [sessionId], references: [sessionId])
+  setWinner      Player[]        @relation("setWinner")
 
   @@map("sets")
 }
 
 model Match {
   matchId        Int             @id @default(autoincrement()) @map("match_id")
+  matchWinner    Player[]        @relation("matchWinner")
```
[the remark this answers is not in the document]
Comment (tydolla00) [reply prisma/schema.prisma:87]: I think we can just omit them all then.

**Review by Shargrove09 — COMMENTED** [review]
Done

**Pull request merged**